packages/shared/src/components/fields/RadioItem.tsx

```diff
@@ -62,7 +62,9 @@ export function RadioItem<T extends string>({
           <span
             className={classNames(
               'flex h-full w-full rounded-full border-2 ',
-              !disabled && 'border-text-tertiary',
+              !disabled
+                ? 'border-text-tertiary'
+                : '!border-surface-disabled !bg-surface-disabled',
```

Comment:
Since you do ternary on the border you don't need important on the second one right?

Reply:
That's because there are some injected styles in the bottom that this is overriding, I'm not even sure the border-text-tertiary is even applied

Reply:
removing it